requirements.txt

```diff
@@ -0,0 +1,4 @@
+black
```

Comment:
should dev requirements be separated?

Reply:
Something like the new `requirements.txt` and `requirements/`?